### PR TITLE
Add better error message for 413 errors in ML

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril.lpz@gmail.com>

#### Description of what you did:
This PR adds better error message for 413 responses

### CODE REVIEW

**Review by petersg83 — CHANGES_REQUESTED** [review]
Is it possible to have a shorter message? It looks like it's too long

Capture d’écran 2020-09-28 à 11 16 35

**Review by petersg83 — APPROVED** [review]
LGTM :)

**Comment by sam-pires** [comment]
spec update: 
text to display "The file is too big"

File: packages/strapi-admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -250,6 +250,7 @@
   "app.utils.add-filter": "Add filter",
   "app.utils.defaultMessage": " ",
   "app.utils.delete": "Delete",
+  "app.utils.errors.file-too-big.message": "File too big",
```
Comment (petersg83) [inline packages/strapi-admin/admin/src/translations/en.json:253]: Missing the "The"

**Review by petersg83 — CHANGES_REQUESTED** [review]

**Review by petersg83 — APPROVED** [review]
LGTM :)

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Comment by sam-pires** [comment]
thanks 👍

**Pull request merged**